AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -460,7 +476,10 @@ - (void)_layoutTextView
   [_textKitComponents.layoutManager invalidateLayoutForCharacterRange:NSMakeRange(0, [_textKitComponents.textStorage length]) actualCharacterRange:NULL];
 
   // When you type beyond UITextView's bounds it scrolls you down a line. We need to remain at the top.
-  [_textKitComponents.textView setContentOffset:CGPointZero animated:NO];
+  // [Question - leotumwattana] Is there a good reason why we are forcing content offset to top? Why not
+  // keep at the current offset of the text view?
+  CGPoint offset = _textKitComponents.textView.contentOffset;
+  [_textKitComponents.textView setContentOffset:offset animated:NO];
```

Comment:
This change seems reasonable to me but I'd rather separate this change out from this PR, since:

- UITextView behavior is weird and sometimes counterintuitive.
- One change per PR makes future debugging easier.
- As written, it seems like this line won't have any effect. It's basically: `textVIew.contentOffset = textView.contentOffset`.